autogen/cluster.tf

```diff
@@ -66,7 +66,15 @@ resource "google_container_cluster" "primary" {
       enabled = pod_security_policy_config.value.enabled
     }
   }
-
+  dynamic "resource_usage_export_config" {
+    for_each = var.resource_usage_export_dataset_id != "" || var.resource_usage_export_network_egress_metering ? [var.resource_usage_export_network_egress_metering] : []
```

Comment:
Not important, but I suggest moving this to locals

Reply:
I think we don't need it in, in case of using single variable.